packages/site/src/assets/badge_runpkg.svg

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="600px" height="595px" viewBox="0 0 600 595" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 61 (101010) - https://sketch.com -->
+    <title>74F9D57D-D524-4A10-9A5E-275106F296DD@1.00x</title>
+    <desc>Created with sketchtool.</desc>
+    <defs>
+        <style type="text/css">
+          @font-face { 
+            font-family: "castledown";
```

Comment:
This doesn’t seem right; isn’t this supposed to use Helvetica? Castledown seems to just be “brute force included” multiple times

Reply:
Good shout... I grabbed this from the Renature docs maybe we need to update it there as well.